TACC/tup-ui
# Add route structure for Projects section

## Overview: Add the routing structure for the Projects section:
- Based on the mockups, the projects listing and the individual project view are separate layouts that don't share UI.
- The [Project Detail View](https://xd.adobe.com/view/cc3dc245-2792-4e10-bdc1-50d2a7d32979-996e/screen/0ecf6806-00b2-4e07-b2b6-ab8b3290c493/) and the [Project Member view](https://xd.adobe.com/view/cc3dc245-2792-4e10-bdc1-50d2a7d32979-996e/screen/30a8fb69-771f-4f00-aa30-98902ca6d2be/) share header/sidebar components, so they are both socketed into outlets in the `<ProjectView/>` component.

## Related:

- [TUP-383](https://jira.tacc.utexas.edu/browse/TUP-383)

## Changes:

## Testing:

1. Go to `http://localhost:8000/dashboard/projects` and you should see this:
<img width="702" alt="image" src="https://user-images.githubusercontent.com/12601812/206291220-f2708cda-296c-4df2-9c40-b56e72a6e134.png">

2. Go to `http://localhost:8000/dashboard/projects/12345` and you should see this:
<img width="708" alt="image" src="https://user-images.githubusercontent.com/12601812/206291368-77af25bd-89a9-49a5-ba93-102e243cd73c.png">

3. Go to `http://localhost:8000/dashboard/projects/12345/testuser` and you should see this:
<img width="1410" alt="image" src="https://user-images.githubusercontent.com/12601812/206291455-7ca28878-0c3f-4cf9-9292-d592a2410be6.png">


## UI:

## Notes:
